docker-compose.yml

```diff
@@ -55,6 +55,7 @@ services:
       - HMPPS_SQS_PROVIDER=localstack
       - HMPPS_SQS_LOCALSTACKURL=http://localstack:4566
       - SERVICES_NDELIUS_INTEGRATION_API_BASE_URL=http://wiremock:8080
+#      - API_CLIENT_SECRET=${API_CLIENT_SECRET}
```

Comment:
Left commented out to easily reintroduce. This will allow us to connect to dev with a local ui